src/main/java/org/terasology/structureTemplates/components/SpawnBlockRegionsComponent.java

```diff
@@ -27,14 +28,41 @@
 /**
  * Fills regions with blocks when a entity with component receives a {@link SpawnStructureEvent}.
  */
-public class SpawnBlockRegionsComponent implements Component{
+public class SpawnBlockRegionsComponent implements Component {
     public List<RegionToFill> regionsToFill = new ArrayList<>();
 
+    public Region3i getBoundingBox() {
+        if (regionsToFill == null) {
+            return null;
```

Comment:
Should the bounding box be of zero size in this case? Not sure if I would expect `null` to be returned for an empty template. Otherwise, add Javadoc to clarify this.